thirdparty/build-thirdparty.sh

```diff
@@ -704,6 +719,7 @@ build_leveldb
 build_brpc
 build_rocksdb
 build_librdkafka
+build_flatbuffers
```

Comment:
Why compile it separately?

Reply:
make[2]: *** No rule to make target `/home/work/project/doris/thirdparty/installed/bin/flatc', needed by `src/arrow/ipc/File_generated.h'.  Stop.
make[1]: *** [src/arrow/ipc/CMakeFiles/metadata_fbs.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
not so familiar with cmake, maybe there are some better solution work for ubuntu14.04

Reply:
What's your cmake version?

Reply:
cmake version 3.16.2 gcc (Ubuntu 7.4.0-1ubuntu1~14.04~ppa1) 7.4.0

Reply:
The cmake is new enough, can you udpate doris to the current master version? I have already bumped up arrow to 0.15.1 version. You can try to see if the problem is there too.

Reply:
-- CMAKE_CXX_FLAGS:  -Wno-noexcept-type  -fdiagnostics-color=always -O3 -DNDEBUG  -Wall -msse4.2 
CMake Error at src/arrow/ipc/CMakeLists.txt:63 (get_target_property):
  get_target_property() called with non-existent target "flatbuffers::flatc".


CMake Error at src/arrow/ipc/CMakeLists.txt:65 (get_target_property):
  get_target_property() called with non-existent target "flatbuffers::flatc".

Reply:
OK, I will merge it

Reply:
en, if ubuntu install flatbuffer already, it also may work, but it may need the true version, so use the compiled flatbuffers in doris thirdparty may ensure always true